porcupine/plugins/highlight/tree-sitter-token-mappings/python.yml

```diff
@@ -6,49 +6,109 @@ dont_recurse_inside:
 
 token_mapping:
   comment: Token.Comment
+  ERROR: Token.Error
```

Comment:
When does this show up?

Reply:
With f-strings it shows up when there's no character after the bang, I now could not find any other places where tree-sitter would yield it

![image](https://user-images.githubusercontent.com/77941087/184417882-85243450-d71f-4ffc-84fa-0614d7944c3a.png)

Reply:
```suggestion
  ERROR: Token.Error  # happens at least for f"{asdf!}" with nothing after the '!' character
```

Reply:
After all I'm not sure whether this is actually useful. Pyflakes will mark it as syntax error anyway.

Reply:
```python
def foo: ...
```
```
type=module text='def foo: ...\n'
  type=ERROR text='def foo:'
    type=def text='def'
    type=identifier text='foo'
    type=: text=':'
  type=expression_statement text='...'
    type=ellipsis text='...'
```

Reply:
If we want to display `ERROR` tokens, we should add `ERROR` to `dont_recurse_inside`. But again, pyflakes will highlight the error anyway and also provide a description for it upon hovering, so maybe we shouldn't handle `ERROR`.